src/jobs/modActionTimers.ts

```diff
@@ -18,7 +18,7 @@ const sql = postgres({
 const currentCases = await sql<[{ guild_id: Snowflake; case_id: number; action_expiration: string }]>`
 	select guild_id, case_id, action_expiration
 	from cases
-	where action_processed = false`;
+	where action_processed = false and action != 7`;
```

Comment:
Why does this need a logic change?

Can't we keep most of the logic that mutes had? So when it would run the stuff to "unmute" the person, we simply remove all the logic code that would remove the role, but still keep the embeds posted.

Reply:
That leading to an issue is probably just a misperception on my end. Since the bot does not allow you to overwrite a timeout, and neither does the native UI (the button is replaced with timeout remove) the "layering" or "extending" of timeouts I feared might be an issue, should indeed not be one.

Expiring of the action could be done manually, which - too - needs to be handled, I'll look into it.

Reply:
On the other hand, the question remains if these are of any use, now that it's handled by discord. Given the shown time points to the past one can infer the timeout being over.

Reply:
I'm not sure I understand the issue, is there a GMU when you "remove" a timeout in the UI?

If its detectable that should be fine?
And like you said, you cannot overwrite it, are you sure? Can you not prolong a timeout via code/the API?

Reply:
And yes, you can always infer that a mute was over based on the old way we did it in v1 too, the additional embed is really just information and serves no real purpose but that, and I think I would like to keep it

Reply:
>  is there a GMU when you "remove" a timeout in the UI?

There are events for when the key changes on the guild member, which is the case on manual timeout and timeout remove. It's not the case when the timeout naturally expires. The member keeps the expired timestamp on the structure.

Reply:
I still don't understand the problem.

Currently the way the log to "remove" a timeout works, is by simply saving the expiry and the worker runs every 1?/5? minutes and then sets the case to be resolved and posts the log.

How is discord implementation changing any of this?